### PR TITLE
docs(aws-lambda) expand on `skip_large_bodies`

### PR DESCRIPTION
**Pull request by murillopaula**

### CODE REVIEW

File: app/_hub/kong-inc/aws-lambda/index.md [hunk 1]
```diff
@@ -175,7 +175,16 @@ params:
       required: false
       default: "`true`"
       description: |
-        An optional value that defines whether Kong should send large bodies that are buffered to disk. To define the threshold for the body size, use the [client_body_buffer_size](https://docs.konghq.com/latest/configuration/#client_body_buffer_size) property. Note that sending large bodies will have an impact on system memory.
+        An optional value that defines whether Kong should send large
+        bodies that are buffered to disk. Note that enabling this
```
Comment (tlblessing) [inline app/_hub/kong-inc/aws-lambda/index.md:179]:
```suggestion
        bodies that are buffered to disk. Note that 
```

repeated phrase

File: app/_hub/kong-inc/aws-lambda/index.md [hunk 2]
```diff
@@ -175,7 +175,16 @@ params:
...
+        enabling this option will have an impact on system memory depending on the number of requests simultaneously in flight at any given point in time and on the maximum size of each of them.
+        Also this option will block all requests being handled by the
+        nginx workers. That may be tens of thousands of other
```
Comment (tlblessing) [inline app/_hub/kong-inc/aws-lambda/index.md:182]:
```suggestion
        nginx workers. That could be tens of thousands of other
```

File: app/_hub/kong-inc/aws-lambda/index.md [hunk 3]
```diff
@@ -175,7 +175,16 @@ params:
...
+        transactions which are not being processed. For small I/O
+        operations, such a delay would generally not be problematic. In
+        cases where the body is in the order of MB, such a delay would
+        cause notable interruptions in request processing. Given all of the downsides of enabling this option, please consider increasing the [client_body_buffer_size](http://nginx.org/en/docs/http/ngx_http_core_module.html#client_body_buffer_size)
```
Comment (tlblessing) [inline app/_hub/kong-inc/aws-lambda/index.md:186]:
```suggestion
        cause notable interruptions in request processing. Given all of the potential downsides resulting from enabling this option, please consider increasing the [client_body_buffer_size](http://nginx.org/en/docs/http/ngx_http_core_module.html#client_body_buffer_size)
```

File: app/_hub/kong-inc/aws-lambda/index.md [hunk 4]
```diff
@@ -175,7 +175,16 @@ params:
       required: false
       default: "`true`"
       description: |
-        An optional value that defines whether Kong should send large bodies that are buffered to disk. To define the threshold for the body size, use the [client_body_buffer_size](https://docs.konghq.com/latest/configuration/#client_body_buffer_size) property. Note that sending large bodies will have an impact on system memory.
+        An optional value that defines whether Kong should send large
+        bodies that are buffered to disk. Note that enabling this
+        enabling this option will have an impact on system memory depending on the number of requests simultaneously in flight at any given point in time and on the maximum size of each of them.
+        Also this option will block all requests being handled by the
+        nginx workers. That may be tens of thousands of other
+        transactions which are not being processed. For small I/O
```
Comment (tlblessing) [inline app/_hub/kong-inc/aws-lambda/index.md:183]:
```suggestion
        transactions that are not being processed. For small I/O
```

https://www.grammarly.com/blog/which-vs-that/

**Review by tlblessing — CHANGES_REQUESTED** [review]
Thanks for pinging me and doing this PR @murillopaula

**Review by tlblessing — APPROVED** [review]
@murillopaula thank you for the PR. LGTM. Not sure what the other reviewers think.

**Review by masakong2 — APPROVED** [review]

**Review by Tieske — APPROVED** [review]

**Pull request merged**